tests/PromiseAdapter/CallbackPromiseAdapter.php

```diff
@@ -17,6 +17,9 @@ public function __construct(array $callbacks)
         $this->callbacks = $callbacks;
     }
 
+    /**
+     * @phpstan-ignore-next-line
```

Comment:
Is this still needed? Should probably be `@return PromiseInterface<T>`, no?

Reply:
Yes, but see comment on `PromiseFulfilledTestTrait`. Would like to have it in, but IMHO not worth it right now.